<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Simple Password Generator</string>
  </property>
  <property name="locale">
   <locale language="English" country="World"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>481</width>
      <height>351</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="locale">
     <locale language="English" country="World"/>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="simple">
     <property name="whatsThis">
      <string>Simple</string>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="locale">
      <locale language="English" country="World"/>
     </property>
     <attribute name="title">
      <string>Simple Generator</string>
     </attribute>
     <widget class="QSlider" name="passwordLength">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>0</y>
        <width>301</width>
        <height>41</height>
       </rect>
      </property>
      <property name="minimum">
       <number>8</number>
      </property>
      <property name="maximum">
       <number>24</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksBelow</enum>
      </property>
      <property name="tickInterval">
       <number>1</number>
      </property>
     </widget>
     <widget class="QCheckBox" name="charNumbers">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>94</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Numbers?</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="charLowerCase">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>151</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>lower case letters?</string>
      </property>
     </widget>
     <widget class="QCommandLinkButton" name="commandLinkButton">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>170</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Execute order 66</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="charUpperCase">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>151</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>upper case letters?</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="charSpecial">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>131</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>special chars?</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="Output">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>240</y>
        <width>311</width>
        <height>41</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
      <property name="placeholderText">
       <string>Generated Password Here</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>91</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Length:</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="displayLength">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>10</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="intValue" stdset="0">
       <number>8</number>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>483</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionReset"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
